commands/example/exampleCommand.js

```diff
@@ -1,17 +0,0 @@
-const { ApplicationCommandType } = require('discord.js');
-module.exports = {
-    name: 'example1',
-    description: "Example command for Officier",
-    options: [
-        {
-            name: 'example-argument',
-            description: 'Add role to a user.',
-            type: 3,
-            required: false,
-        }
-    ],
-    type: ApplicationCommandType.ChatInput,
-    run: async (client, interaction) => {
-        return interaction.reply({ content: "Example command" })
-    }
-};
```

Comment:
To było po to by ktoś kto robi forka miał łatwy example

Reply:
I też teraz to może mi popsuć całego PR z modułem od muzyki przez błędy które wystąpią z powodu
```
"
```

Reply:
Takie rzeczy mona wrzucić do CONTRIBUTING.md, albo jakiegoś innego directory

Reply:
Jeszcze nie zrobiłeś tego PR

Reply:
Dosłownie 2/3 polecenia pozostały i moduł jest gotowy

Reply:
Okej, to jak zrobisz PR to ci porozwiązuje conflicty